app/views/landing-pages/parents/PageParents.vue

```diff
@@ -34,30 +34,24 @@
         <div class="row">
           <div class="col-lg-12 trust-logos">
             <div class="flex-spacer">
-              <img srcset="/images/pages/parents/cse_top_pick@1x.png 1x,
-                                        /images/pages/parents/cse_top_pick@2x.png 2x,
-                                        /images/pages/parents/cse_top_pick@3x.png 3x"
-                  src="/images/pages/parents/cse_top_pick@1x.png"
+              <img
+                  src="/images/pages/parents/cse_top_pick.png"
                   class="cse-top-pick"
               />
             </div>
 
             <div class="flex-spacer">
               <div class="cs-for-all-container">
-                <img srcset="/images/pages/parents/cs_for_all_member@1x.png 1x,
-                                        /images/pages/parents/cs_for_all_member@2x.png 2x,
-                                        /images/pages/parents/cs_for_all_member@3x.png 3x"
-                    src="/images/pages/parents/cs_for_all_member@1x.png"
+                <img 
+                  src="/images/pages/parents/cs_for_all_member.png"
                 />
               </div>
             </div>
 
             <div class="flex-spacer">
               <div class="codie-logo-container">
-                <img srcset="/images/pages/parents/2017_codie_award@1x.png 1x,
-                                        /images/pages/parents/2017_codie_award@2x.png 2x,
-                                        /images/pages/parents/2017_codie_award@3x.png 3x"
-                    src="/images/pages/parents/2017_codie_award@1x.png"
+                <img
+                    src="/images/pages/parents/2017_codie_award.png"
```

Comment:
Removing the img optimization without updating the static assets causes these to break:
![image](https://user-images.githubusercontent.com/15080861/105088067-3fd8fa00-5a50-11eb-94fd-46ae170c04d0.png)

I'm a little confused why tis was done. I believe the html changes can be reverted, as we only need the CSS changed.

Reply:
Images was missing so I committed new images and removed old images